src/Log.php

```diff
@@ -277,9 +281,17 @@ public static function history($items_id, $itemtype, $changes, $itemtype_link =
             'date_mod'          => $date_mod,
             'id_search_option'  => $id_search_option,
             'old_value'         => $old_value,
-            'new_value'         => $new_value
+            'new_value'         => $new_value,
         ];
 
+        if (
```

Comment:
Please add a comment to explain why.
```suggestion
        // This condition prevents errors during migrations from older GLPI versions,  
        // where some steps write logs before the new columns are added.
        if (
```